Lib/uuid.py

```diff
@@ -412,20 +420,71 @@ def _find_mac(command, args, hw_identifiers, get_index):
         pass
     return first_local_mac or None
 
+# Keyword is only on firstline - values on remaining lines
+def _find_mac_nextlines(command, args, hw_identifiers, f_index):
+    first_local_mac = None
+    mac = None
+    try:
+        proc = _popen(command, *args.split())
+        if not proc:
+            return None
+        with proc:
+            keywords = proc.stdout.readline().rstrip().split()
+            try:
+                i = keywords.index(hw_identifiers)
+            except ValueError:
+                return None
+            # we have the index (i) into the data that follows
+            for line in proc.stdout:
+                try:
+                    values = line.rstrip().split()
+                    value = values[f_index(i)]
+                    if len(value) == 17 and value.count(_MAC_DELIM) == 5:
+                        mac = int(value.replace(_MAC_DELIM, b''), 16)
+                    # (Only) on AIX the macaddr value given is not prefixed by 0, e.g.
+                    # en0   1500  link#2      fa.bc.de.f7.62.4 110854824     0 160133733     0     0
+                    # not
+                    # en0   1500  link#2      fa.bc.de.f7.62.04 110854824     0 160133733     0     0
+                    elif _AIX and  value.count(_MAC_DELIM) == 5:
```

Comment:
And this can change to `_MAC_OMITS_LEADING_ZEROES` rather than being a direct reference to `_AIX`